Doc/library/argparse.rst

```diff
@@ -1122,6 +1122,8 @@ is used when no command-line argument was present::
    >>> parser.parse_args([])
    Namespace(foo=42)
 
+For other nargs_ values, the argument is treated as a required argument and
+the default_ value is ignored.
```

Comment:
This looks misleading, because it is not clear that this only relates to positional arguments.

On other hand, default is also has no effect for required optional arguments.

We could say that the default value is ignored for required arguments (there may be a reference to the `required` attribute).